docs/src/modules/java/pages/views.adoc

```diff
@@ -486,7 +486,9 @@ Or come from fields in the request message:
 SELECT * FROM customers OFFSET :start_from LIMIT :max_customers
 ----
 
-Note: `OFFSET` and `LIMIT` cannot be used in queries containing `OR`.
+Note: Using numeric offsets can lead to missing or duplicated entries in the result if entries are added to or removed from the view between requests for the pages. The token based paging handles this better.
```

Comment:
Perhaps we should just describe in what cases entries are duplicated/missing without pointing to token-based paging? token-based paging is not *much* better, after all...

Reply:
Lets' remove the token based reference, but not sure I agree we should spend doc-space detailing exact scenarios.